app/models/course/assessment/question/text_response.rb

```diff
@@ -2,9 +2,15 @@
 class Course::Assessment::Question::TextResponse < ApplicationRecord
   acts_as :question, class_name: Course::Assessment::Question.name
 
+  DEFAULT_MAX_ATTACHMENT_SIZE_MB = 1024
+
   validates :max_attachments, numericality: { only_integer: true, greater_than_or_equal_to: 0,
                                               less_than_or_equal_to: 50 },
                               presence: true
+  validates :max_attachment_size, numericality: { only_integer: true, greater_than_or_equal_to: 1,
```

Comment:
good. can we do something similar for max_attachments in line 7 as well?

Reply:
I think those 2 are already similar?

Reply:
done